statemachine/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,14 @@
+{
+  "StateMachineName": "{{ContractTestStateMachineName}}",
+  "RoleArn": "{{ContractTestStateMachineRole}}",
+  "StateMachineType": "STANDARD",
```

Comment:
minor: do we have any coverage on express? would be good to add some variability to tests to cover more areas

Reply:
Added test inputs for EXPRESS. Tested that the contract tests pass with them.